### ISSUE TITLE
Empty email for new project links

### ISSUE THREAD

**Issue by gondzo** [topcoder-platform/tc-notifications#81]
New project links email doesn't have any data (user name is empty, links aren't listed)

![image](https://user-images.githubusercontent.com/3051646/51247684-32cf2f00-198e-11e9-8fab-25250bcc01bc.png)

**Comment by vikasrohit** [comment]
@gondzo Yes, we identified this earlier and applied a patch for this. Are you still facing the same issue?

**Comment by gondzo** [comment]
yes, got the email an hour ago.

**Comment by vikasrohit** [comment]
Thanks @gondzo for bringing to notice, applied patch for it. Let us know if you still see the same.
